substrate/client/transaction-pool/tests/fatp.rs

```diff
@@ -2328,7 +2328,12 @@ fn fatp_ready_light_long_fork_retracted_works() {
 	let header02b = api.push_block_with_parent(header01b.hash(), vec![xt1.clone()], true);
 	let header03b = api.push_block_with_parent(header02b.hash(), vec![xt2.clone()], true);
 
+	// Returns the most recent view (`header01a`) ready transactions set.
 	let mut ready_iterator = pool.ready_at_light(header03b.hash()).now_or_never().unwrap();
+	assert!(ready_iterator.next().is_some());
+	assert!(ready_iterator.next().is_some());
+	assert!(ready_iterator.next().is_some());
+	assert!(ready_iterator.next().is_some());
```

Comment:
Can we do sth like this:
```suggestion
	assert_eq!(ready_iterator.count(), 4);	
```